tests/integration/test_scmrepository_view.py

```diff
@@ -117,7 +65,13 @@ def test_filtered_on_org_and_repo_name(
         assert parsed[0]["repository_name"] == "sample"
 
     def test_filtered_list_non_existing_credential(
-        self, client, logged_in_user, application, scm_pipeline_run, another_application, another_scm_pipeline_run
+        self,
+        client,
+        logged_in_user,
+        my_application,
+        my_scm_pipeline_run,
+        my_other_application,
+        another_scm_pipeline_run,
```

Comment:
Why does black only decide to do this now? Did we not lint the tests properly before?

Reply:
because of application -> my_application and others